W-14379658 Update imp-implementing-apis.adoc (#238)
mulesoft/docs-code-builder: modules/ROOT/pages/imp-implementing-apis.adoc

[[import-spec-into-project]]
* xref:imp-implement-apis.adoc#import-spec-into-project[Import an API spec] from Exchange to scaffold the API in an existing integration project.
* xref:imp-implement-local-apis.adoc[Scaffold an API] into an implementation project without first publishing it to Anypoint Exchange, enabling iterative design and implementation.

> After scaffolding an API specification, find Mule application development processes described in xref:int-developing-integrations.adoc[]. To understand how an integration project can implement an API specification, see the tutorial xref:tut-af-implement-am-flights-api.adoc[].

Reviewer: ```suggestion
After scaffolding an API specification, you are ready to develop your Mule application as described in xref:int-developing-integrations.adoc[]. To understand how an integration project can implement an API specification, see the tutorial xref:tut-af-implement-am-flights-api.adoc[].
```

> After scaffolding an API specification, you are ready to develop your Mule application as described in xref:int-developing-integrations.adoc[]. To understand how an integration project can implement an API specification, see the tutorial xref:tut-af-implement-am-flights-api.adoc[].

Author: ```suggestion
After scaffolding an API specification into an interface, you are ready to develop, test, and deploy your Mule application as described in xref:int-developing-integrations.adoc[]. To understand how an integration project can implement an API specification, see the tutorial xref:tut-af-implement-am-flights-api.adoc[].
```
